app/src/main/java/us/shandian/giga/io/StoredFileHelper.java

```diff
@@ -275,7 +275,7 @@ public boolean create() {
             if (!docTree.canRead() || !docTree.canWrite()) return false;
             try {
                 docFile = createSAF(context, srcType, srcName);
-                if (docFile == null || docFile.getName() == null) return false;
+                if (docFile.getName() == null) return false;
```

Comment:
Are you completely sure about removing this null check?

Reply:
Yes, let me annotate the method as `NonNull`